terracotta/config.py

```diff
@@ -105,6 +111,9 @@ class SettingSchema(Schema):
         validate=validate.OneOf(['nearest', 'linear', 'cubic', 'average'])
     )
 
+    ALLOWED_ORIGINS_METADATA = fields.String()
```

Comment:
Shouldn't these be more like list-of-string?

Reply:
Could be, too. OK then.

Reply:
I think list of string is less ambiguous, so we should support that.